lib/sequenceserver/blast.rb

```diff
@@ -104,6 +103,10 @@ def pre_process(params)
         params[:sequence].strip! unless params[:sequence].nil?
       end
 
+      def blast_formatter(rfile_path, ofmt, specifiers = nil)
+        `blast_formatter -archive '#{rfile_path}' -outfmt '#{ofmt} #{specifiers}'`
```

Comment:
A good thing about specifiers here is that wrong values are happily ignored. So if I pass `specifiers  = "qframe random none"`, then output will contain only of `qframe`.